### DISCUSSION TITLE
Old school menu bar from back in the day

### DISCUSSION THREAD

**Discussion by korvl38** [runelite/runelite#16559] [Ideas]
Little cosmetic suggestion and pretty much useless, but this is peak nostalgia for me. Talking about the top bar with world select, rules, etc. Is it possible?
![image](https://user-images.githubusercontent.com/101873426/236601303-67d9981f-5094-4c8a-a0a3-556e121ec7a4.png)

**Comment by korvl38** [comment]
everything I suggest no one wants to say anything just ignore?

**Reply by Felanbird** [reply]
Would you really prefer if people commented `no that's a bad idea`?

**Reply by korvl38** [reply]
you are asking If I prefer to be heard rather than completely ignored? what do you think
